Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -929,7 +929,7 @@ public void SetState(string key, float value, bool reliable = false)
             {
                 if (IsRunningInBrowser())
                 {
-                    SetPlayerStateFloatByPlayerId(id, key, value.ToString(), reliable);
+                    SetPlayerStateFloatByPlayerId(id, key, value.ToString(CultureInfo.InvariantCulture), reliable);
```

Comment:
Is this done everywhere where `ToString` is used? If yes, has it been tested to not have any negative effects?